lib/login.dart

```diff
@@ -1,138 +1,84 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/gestures.dart';
-import 'package:frontend/input_field.dart';
+import 'package:provider/provider.dart';
 import 'main.dart';
+import 'custom_input_field.dart';
+import 'animation_route.dart';
 
-class LoginWidgetState extends State<LoginWidget> {
+class LoginScreen extends StatefulWidget {
+  const LoginScreen({super.key});
 
-  bool _loginUI = true;
-  // Switches between Login and Register UI
-  void _toggleDisplayedUI() {
-    setState(() {
-      _loginUI = !_loginUI;
-    });
-  }
+  @override
+  LoginScreenState createState() => LoginScreenState();
+}
+
+class LoginScreenState extends State<LoginScreen> {
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
         appBar: AppBar(
-          backgroundColor: const Color.fromARGB(179, 85, 85, 85),
+          backgroundColor: const Color.fromARGB(255, 30, 30, 30),
```

Comment:
can't you do any form of global styling? like have a default color palette for the entire frontend?

Reply:
yes, see new changes